<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>770</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_11">
     <item row="1" column="0">
      <widget class="QStackedWidget" name="Menu_StackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="NProcesosQSW_3">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>410</y>
           <width>401</width>
           <height>111</height>
          </rect>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="1" column="1">
           <widget class="QPushButton" name="Aceptar_Botton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Aceptar</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="NP_SpinBox">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="accelerated">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Elige el número de procesos a ingresar:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="PictureMenu_label">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>150</x>
           <y>50</y>
           <width>461</width>
           <height>321</height>
          </rect>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/rcs/Images/carpeta.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="whatsThis">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/rcs/Images/carpeta.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="styleSheet">
          <string notr="true">img src=&quot;:/rcs/Images/carpeta.png&quot;</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="Resource.qrc">:/rcs/Images/carpeta.png</pixmap>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="IProcesosQSW_3">
        <layout class="QGridLayout" name="gridLayout_13">
         <item row="0" column="0">
          <widget class="QGroupBox" name="InfoProcQGB_3">
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_14">
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="NoProceso_Label">
                <property name="text">
                 <string>Ingresa los datos del proceso No:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="NoProceso_LineEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="CapturaProceso_Botton">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Aceptar</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QGroupBox" name="groupBox_6">
              <property name="title">
               <string/>
              </property>
              <layout class="QGridLayout" name="gridLayout_15">
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_12">
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <item>
                    <widget class="QLabel" name="NomProg_Label">
                     <property name="text">
                      <string>Nombre del Programador:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="Operacion_Label">
                     <property name="text">
                      <string>Operación a Realizar:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="TiempoMax_Label">
                     <property name="text">
                      <string>Tiempo máximo estimado:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="IDProc_Label">
                     <property name="text">
                      <string>Identificador del Proceso:</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item row="0" column="1">
                <layout class="QVBoxLayout" name="verticalLayout_8">
                 <item>
                  <widget class="QLineEdit" name="NomProg_LineEdit">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>30</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_13">
                   <item>
                    <widget class="QSpinBox" name="OperacionNo1_SpinBox">
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                     <property name="buttonSymbols">
                      <enum>QAbstractSpinBox::NoButtons</enum>
                     </property>
                     <property name="minimum">
                      <number>0</number>
                     </property>
                     <property name="maximum">
                      <number>999999999</number>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="OperacionSigno_ComboBox">
                     <property name="layoutDirection">
                      <enum>Qt::LeftToRight</enum>
                     </property>
                     <item>
                      <property name="text">
                       <string>+</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>-</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>*</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>/</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>%</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="OperacionNo2_SpinBox">
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                     <property name="buttonSymbols">
                      <enum>QAbstractSpinBox::NoButtons</enum>
                     </property>
                     <property name="maximum">
                      <number>999999999</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="TiempoMax_SpinBox">
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="buttonSymbols">
                    <enum>QAbstractSpinBox::NoButtons</enum>
                   </property>
                   <property name="minimum">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <number>999</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="IDProceso_SpinBox">
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                   <property name="buttonSymbols">
                    <enum>QAbstractSpinBox::NoButtons</enum>
                   </property>
                   <property name="correctionMode">
                    <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
                   </property>
                   <property name="minimum">
                    <number>0</number>
                   </property>
                   <property name="maximum">
                    <number>999999999</number>
                   </property>
                   <property name="value">
                    <number>1</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="PictureMenu2_label">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="Resource.qrc">:/rcs/Images/carpeta.png</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="EjecProcesosQSW_3">
        <widget class="QTableWidget" name="LoteEjecucion_Table">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>270</y>
           <width>271</width>
           <height>411</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>180</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Nombre</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>TME</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
        <widget class="QLabel" name="NoLotesP_Label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>291</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Número de lotes pendientes:</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="NoLotesP_LineEdit">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>10</y>
           <width>81</width>
           <height>24</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="LoteEjecucion_Label">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>240</y>
           <width>181</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Procesos Pendientes:</string>
         </property>
        </widget>
        <widget class="QLabel" name="ProcesosTerm_Label">
         <property name="geometry">
          <rect>
           <x>450</x>
           <y>240</y>
           <width>211</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Procesos terminados:</string>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>761</width>
           <height>161</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="LoteAct_Label">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>Proceso Actual:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="LoteActID_Label">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>ID:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LoteActID_LineEdit">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>24</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="LoteActNombre_Label">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>Nombre:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LoteActNombre_LineEdit">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>24</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="LoteActOperacion_Label">
              <property name="maximumSize">
               <size>
                <width>110</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Operación:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LoteActOperacion_LineEdit">
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>24</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="LoteActTiempoAct_Label">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>40</height>
               </size>
              </property>
              <property name="text">
               <string>TME:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LoteActTiempoAct_LineEdit">
              <property name="maximumSize">
               <size>
                <width>180</width>
                <height>24</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="LoteActTiempoRes_Label">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>TR:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LoteActTiempoRes_LineEdit">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>24</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="LoteActTiempoTra_Label">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>TT:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LoteActTiempoTra_LineEdit">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>24</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="Contador_Label">
         <property name="geometry">
          <rect>
           <x>560</x>
           <y>10</y>
           <width>121</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Contador:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLineEdit" name="Contador_LineEdit">
         <property name="geometry">
          <rect>
           <x>680</x>
           <y>10</y>
           <width>81</width>
           <height>24</height>
          </rect>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QTableWidget" name="ProcesosTerm_Table">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>270</y>
           <width>461</width>
           <height>411</height>
          </rect>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Operacion</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Resultado</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Lote</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
